Assets/MixedRealityToolkit-SDK/Features/Diagnostics/CpuUseTracker.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Diagnostics;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.DiagnosticsSystem
+{
+    internal class CpuUseTracker
```

Comment:
Do we want to make this internal, or just sealed?

Reply:
sealed indicates that code (including the toolkit) cannot extend the class... I tend to fall on the side of internal.